pages/user/order/index.tsx

```diff
@@ -33,19 +35,37 @@ const Orders: React.FC = observer(() => {
       return (
         <div key={order.key} className='my-3'>
           <OrderItem onOrder={orderFood} order={order} />
+
         </div>
       );
     });
 
   return (
-    <div>
-      <div className='text-sm'>
+    <div className="bg-red-200">
+      <div className='text-sm '>
         Ordering as{' '}
         <span className='font-bold'>{userStore.userInfo?.username}</span>
       </div>
+      <Card 
```

Comment:
just use card component on my #28 . There is no need to use 'items' and 'childrenClassName' as props of card component.